composer.json

```diff
@@ -12,17 +12,17 @@
         }
     ],
     "require": {
-        "php": ">=5.6.0",
-        "laravel/framework": "~5.0",
+        "php": ">=7.2",
+        "laravel/framework": "^6.0",
         "ua-parser/uap-php": "~3.8",
         "league/pipeline": "~0.1",
         "mobiledetect/mobiledetectlib": "~2.0",
         "jaybizzle/crawler-detect": "~1.0",
         "piwik/device-detector": "~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~5.0 || ~6.0 || ~7.0",
-        "orchestra/testbench": "~3.0",
+        "phpunit/phpunit": "~8.0",
```

Comment:
why are these dev dependency changes required?

Reply:
Because Laravel needs testbench 3.9 (or 4) and testbench needs phpunit 8. 
Will change it to:
`
"phpunit/phpunit": "~5.0 || ~6.0 || ~7.0 || ~8.0",
        "orchestra/testbench": "~3.0",
`